internal/integration_test/bench/bench_test.go

```diff
@@ -44,11 +46,21 @@ func BenchmarkInitialization(b *testing.B) {
 		runInitializationBench(b, r)
 	})
 
+	b.Run("interpreter-multiple", func(b *testing.B) {
```

Comment:
hopefully at some point we can have interpreter perform well enough to not already know the answer of benchmarks (too slow ;))